MetaMask/metamask-extension
# UX: Multichain: Focus on test networks when setting is flipped on

## Explanation

This PR ensures that the test networks are scrolled to when the setting gets flipped on.

## Screenshots/Screencaps

https://user-images.githubusercontent.com/46655/236033415-c9557717-f11d-4741-9fc9-1e74a02e61c7.mov

## Manual Testing Steps

1.  Open the network picker with test networks turned off
2. Turn them on
3. See the test networks get focused upon

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
